app_feup/lib/controller/local_storage/app_exams_database.dart

```diff
@@ -1,19 +1,38 @@
 import 'dart:async';
 import 'package:uni/controller/local_storage/app_database.dart';
 import 'package:uni/model/entities/exam.dart';
+
+import 'package:path/path.dart';
 import 'package:sqflite/sqflite.dart';
+import 'package:sqflite_migration/sqflite_migration.dart';
 
 /// Manages the app's Exams database.
 /// 
 /// This database stores information about the user's exams.
 /// See the [Exam] class to see what data is stored in this database.
 class AppExamsDatabase extends AppDatabase {
+
+  static final createScript =
```

Comment:
Should be private, pls put an underscore on the variable name.

Reply:
Changed it to 'create_Script'. Realized now that it probably should have been 'create_script'. Is this how you asked it to be?

Reply:
On Dart variables need to start with an underscore in order to be private, so Andre was probably suggesting something like `_createScript`

Reply:
I've corrected this variable's name